File: cpp/fizzbuzz/fizzbuzz.cpp
#include <iostream>
bool is_fizz(int i);
bool is_buzz(int i);

int main(){
    for(int i = 1; i <= 30 ; i++){
        if(is_fizz(i) && is_buzz(i)){
            std::cout << "fizzbuzz\n";
        }
        else if(is_fizz(i)){
            std::cout << "fizz\n";
        }
        else if(is_buzz(i)){
            std::cout << "buzz\n";
        }
        else{
            std::cout << i << '\n';
        }
    }
}

bool is_fizz(int i){
    if( i % 3 == 0){
        return true;
    }
    else{
        return false;
    }
}

bool is_buzz(int i){
    if( i % 5 == 0){
        return true;
    }
    else{
        return false;
    }
}

File: python/fizzbuzz/fizzbuzz.py
def isFizz(num: int) -> bool:
    if (num % 3 == 0):
        return True
    else:
        return False


def isBuzz(num: int) -> bool:
    if (num % 5 == 0):
        return True
    else:
        return False


def main():
    for i in range(1, 30):
        if (isFizz(i)) & (isBuzz(i)):
            print(f'fizzbuzz')
        elif (isFizz(i)):
            print(f'fizz')
        elif (isBuzz(i)):
            print(f'buzz')
        else:
            print(f'{i}')


if __name__ == "__main__":
    main()
